src/data.js

```diff
@@ -1,33 +1,147 @@
-const listArray = [
-  {
-    description: 'Complete To-Do list',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Take a walk',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Watch a movie',
-    completed: false,
-    index: 2,
-  },
-];
-
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] As requested in the previous review kindly remove this eslint disable line. You can simply move the add function to the top of the module and next you should avoid calling functions in a loop. That is a first function calls a second function and then that second function still calls the first function again, which would throw this eslint error. Kindly fix it.